incubator/nodejs-express/README.md

```diff
@@ -1,79 +1,65 @@
+Health-checking enables the cloud platform to determine `readiness` (is your application ready to receive requests?) and `liveness` (is your application alive or does it need to be restarted?) of your application.
 
 The [@cloudnative/cloud-health](https://github.com/CloudNativeJS/cloud-health) module is a core library that provides health checking and gives you more control over application lifecycle management (startup, running, shutdown). The [@cloudnative/health-connect](https://github.com/CloudNativeJS/cloud-health-connect) module exposes the health-check information on readiness/liveness endpoints to help the cloud platforms manage your application.
 
-You can override or enhance the following endpoints by configuring your own health checks in `app.js`. If you do not specify a health endpoint thes endpoints will be available for you to use:
+ These endpoints will be available for you to use:
+
+- Readiness endpoint: http://localhost:3000/ready
+- Liveness endpoint: http://localhost:3000/live
+- Health check endpoint: http://localhost:3000/health
 
-- Readiness Check: http://localhost:3000/ready
-- Liveness Check: http://localhost:3000/live
-- Health Check: http://localhost:3000/health
+You can override or enhance the following endpoints by configuring your own health checks in your application.
 
-### Application metrics
+## Application metrics
 
 Enable powerful monitoring for your distributed application and configure rule-based alerting using Prometheus open source project. This is vital for diagnosing problems and ensuring the reliability of your application.
 
 The [appmetrics-prometheus](https://github.com/CloudNativeJS/appmetrics-prometheus) module will collect a wide range of resource-centric (CPU, memory) and application-centric (HTTP request responsiveness) metrics from your application, and then expose them as multi-dimensional time-series data through an application endpoint for Prometheus to scrape and aggregate.
 
-This stack also comes with prometheus metrics, which has been preconfigured to work with your applicaiton. You will not be able to override this endpoint:
+This stack also comes with Prometheus metrics, which has been preconfigured to work with your application. You will not be able to override this endpoint:
 
-- Metrics: http://localhost:3000/metrics
+- Metrics endpoint: http://localhost:3000/metrics
 
 ## Templates
 
-A template is what you will use as a template to begin your developement. When initializing your project you have the option to choose between using the hello-world application or the hello-express application. This stack consists of two nodejs-express templates, A hello-world and a hello-express template.
+Templates are used to create your local project and start your development. When initializing your project you will be provided with the simplest Node.js Express application you can write.
```

Comment:
Can you move templates up to the top as per the other Stack READMEs?

Reply:
In this case the stack includes these capabilities not the template.. hence the template section is further down.

Reply:
this ones had more capabilities in the stack. hence the first sections looks bigger..

Reply:
Ah ok, it wasn't obvious to me that the stack contained those capabilities.